mappings/net/minecraft/entity/passive/LlamaEntity.mapping

```diff
@@ -2,8 +2,10 @@ CLASS any net/minecraft/entity/passive/LlamaEntity
 	FIELD bM ATTR_STRENGTH Lpw;
 	FIELD bO ATTR_VARIANT Lpw;
 	METHOD b createChild (Lahj;)Lany;
+	METHOD ek isTraderLlama ()Z
```

Comment:
Why not just `isTrader`? It's in the llama class, so it's clear it's a llama, we just need to know if the llama is also a trader.

Reply:
I fear trader might imply the wandering trader itself.

Reply:
Done.